.circleci/config.yml

```diff
@@ -41,6 +41,7 @@ jobs:
             sudo ./aws/install
       - run: aws --version
       - run: aws s3 sync site/ s3://semgrep-docs && aws cloudfront create-invalidation --distribution-id E1PHOFPGTV8DCC --paths '/docs/*'
```

Comment:
```suggestion
      - run: aws s3 sync site/ s3://semgrep-docs/docs && aws cloudfront create-invalidation --distribution-id E1PHOFPGTV8DCC --paths '/docs/*'
```

Reply:
I added the line below to sync to /docs cause I wasn't sure if something else was relying on it being in root?

Reply:
ah makes sense. thanks